### ISSUE TITLE
buildcraft - Not Working

### ISSUE THREAD

**Issue by newborn1000** [MyEssentials/MyTown2-Protections#57]
- [ ] Filler
- [ ] Builder

**Comment by krwminer** [comment]
Please give more detail on these. What it is not working, being able to use them in your own town or breaking into protected area? How you set them up, etc...

**Comment by newborn1000** [comment]
They are able to break into a protected area according to the mod who tested this. We've ascertained that they can be placed outside of the protection zone and with landmarks, can be used to fill the town.

Also the builder uses blueprints much the same way, Wiki links to them here: 

http://minecraftbuildcraft.wikia.com/wiki/Filler
http://minecraftbuildcraft.wikia.com/wiki/Builder

Interestingly the Quarry which also seems to use landmarks, can't link to the landmarks to get the larger area filled. Presumably this is the way the protection works to counteract Quarries.

**Comment by krwminer** [comment]
I tried this out this morning before work. I used a case where I overlapped a corner of a protected area with the landmarks. After placing the landmarks and activating the laser markers I placed a Filler next to it. The Filler broke along with the landmarks. So... Lets figure out what is different between our setups so we can get to the bottom of it. :)

What version of BuildCraft are you using? Are you also using BuildCompat? How did you test the protect? (Overlap, fully encompass the protected area, etc.)

Thanks for your help.
